### PR TITLE
Update custom_output.md

### PR DESCRIPTION
**Pull request by beckettsean**

### CODE REVIEW

**Comment by beckettsean** [comment]
@rossmcdonald @nathanielc

**Comment by rossmcdonald** [comment]
These changes look good to me 👍

File: content/kapacitor/v1.0/about_the_project/custom_output.md [hunk 1]
```diff
@@ -99,15 +97,14 @@ type HouseDBOutNode struct {
...
 Next we need a consistent way to create a new instance of our node.
 But to do so we need to think about how this node connects to other nodes.
-Since we are an output node as far as Kapacitor is concerned we are the end of the pipeline and we do not provide any outbound edges.
-HouseDB is a flexible datastore and can store data in batches or as single data points,
-as a result we do not care what type of data the HouseDBOutNode node receives.
-With that in mind we can define a function to create a new HouseDBOutNode.
-Add this function to the end of the `housedb_out.go` file.
+Since we are an output node as far as Kapacitor is concerned this is the end of the pipeline. 
+We will not provide any outbound edges, the graph ends on this node.
+Out imaginary HouseDB is flexible and can store data in batches or as single data points.
```
Comment (nathanielc) [inline content/kapacitor/v1.0/about_the_project/custom_output.md]: Our*

File: content/kapacitor/v1.0/about_the_project/custom_output.md [hunk 2]
```diff
@@ -493,8 +491,7 @@ func newMyCustomNode(e EdgeType, n Node) *MyCustomNode {
 }
 ```
 
-Second it is possible to define a method that sets fields on a pipeline Node and just return the same instance
-in order to create a `property method`.
+The second difference is that it is possible to define a method that sets fields on a pipeline Node and just return the same instance in order to create a `property method`.
```
Comment (nathanielc) [inline content/kapacitor/v1.0/about_the_project/custom_output.md]: just return -> returns

**Comment by nathanielc** [comment]
@beckettsean A few nits, otherwise looks great thanks!

**Comment by beckettsean** [comment]
nits fixed, thanks @rossmcdonald and @nathanielc for reviewing!

**Pull request merged**